front/src/pages/room/style.tsx

```diff
@@ -0,0 +1,102 @@
+import styled from 'styled-components';
+
+export const Wrapper = styled.div`
+  border: 1px solid black;
+  box-sizing: border-box;
+  width: 360px;
+  height: 100%;
+  -webkit-box-shadow: 0px 1px 16px 2px #000000;
+  box-shadow: 0px 1px 16px 2px #000000;
+`;
+export const NameField = styled.div`
+  display: flex;
+  justify-content: space-between;
+  border-bottom: 1px solid black;
+  width: 100%;
+  text-align: center;
+`;
+export const BackBtn = styled.button`
+  border: none;
+  background: none;
+`;
+export const ChatBox = styled.div`
+  box-sizing: border-box;
+  width: 100%;
+  height: 100%;
+`;
+
+export const MessageBox = styled.div`
+  overflow: scroll;
+
+  width: 100%;
+  height: 500px;
+`;
+
+export const NameBox = styled.span<{ me?: boolean }>`
+  display: flex;
+  justify-content: ${(props) => (props.me ? 'flex-end' : null)};
```

Comment:
destructuring 해주는게 어떨까요?! 그리고 me가 없을 때 설정을 하지 않는 경우에는 && 연산자 사용해도 좋을 것 같아요!
```suggestion
  justify-content: ${({me}) => (me && 'flex-end')};
```

Reply:
네네 더 깔끔하네용 👍